deploy.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+GITHUB_TAG=${GITHUB_REF/refs\/tags\//}
```

Comment:
tohle funguje?  tak asi uvidime :-)

Reply:
joo, funguje - používáme to tak všude, např:
https://github.com/keboola/db-extractor-mssql/blob/master/deploy.sh#L4